html_app/ui/experiment_setup.soy

```diff
@@ -141,8 +141,22 @@ Experiment active area
     {/if}
     </div>
     <div class="scb_s_experiment_setup_new_set_up">
-    	<div class="scb_s_experiment_setup_instructions" aria-label='Setup Instructions' role='textbox'>
-            	{$t.experiment_setup|noAutoescape}
+        <div class="scb_s_experiment_setup_instructions" aria-label='Setup Instructions' role='textbox'>
+            {if $t.experiment_setup}
+                {$t.experiment_setup|noAutoescape}
+            {else}
+                <ul role='list' class='scb_assignment_specific_tufts_experiment_setup'>
+                    <li role='listitem'>
+                        To setup your experiment, select <b role='presentation' >Add Samples</b>
```

Comment:
It looks like Add Samples and in run together in the browser:

> To setup your experiment, select Add Samplesin the experimental set-up table below. 

I think if you move the word `in` from the previous line onto the end of this line it will fix it